ibm_catalog.json

```diff
@@ -9,28 +9,44 @@
           "target_terraform",
           "terraform",
           "reference_architecture",
-          "solution"
+          "solution",
+		  "Enterprise applications"
```

Comment:
please fi the indentation

Reply:
also this is not a valid tag. Please re-check what I said in https://github.com/terraform-ibm-modules/terraform-ibm-mas/pull/79#discussion_r1587460134

FYI category tags can be got from running `ibmcloud catalog filter options --all`